src/App.jsx

```diff
@@ -1,33 +1,45 @@
 import './App.scss';
+import { useState } from 'react';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
-export const App = () => (
-  <div className="page">
-    <div className="page-content">
-      <div className="box">
-        <div className="field">
-          {/* eslint-disable-next-line jsx-a11y/label-has-associated-control */}
-          <label htmlFor="search-query" className="label">
-            Search movie
-          </label>
+export const App = () => {
+  const [query, setQuery] = useState('');
+  const [visibleMovies, setVisiblesMovies] = useState(moviesFromServer);
 
-          <div className="control">
-            <input
-              type="text"
-              id="search-query"
-              className="input"
-              placeholder="Type search word"
-            />
+  const eventInput = event => setQuery(event.target.value.trim().toLowerCase());
```

Comment:
```suggestion
  const handleQueryChange = event => setQuery(event.target.value.trim().toLowerCase());
```
Pay attention to naming conventions
https://javascript.plainenglish.io/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364